### ISSUE TITLE
place=farm doesn't show on search

### ISSUE THREAD

**Issue by AntMadeira** [openstreetmap/id-tagging-schema#63]
Hi there.

The preset place=farm doesn't show when searching, although the correct icon and translation appears when manually inserted. Is this on purpose or an error?

Regards.

**Comment by tyrasd** [comment]
Apparently, it was made non-searchable because too many users mistook it for other farm features, see https://github.com/openstreetmap/iD/commit/eb62e22a375aa0fa689befa9b1c3560dcbf1e8bf. I can see that this tag has indeed quite a bit of overlap with other tags (landuse=farmyard, building=farm, place=isolated_dwelling, etc.), so there really is a potential for confusion. :thinking: 

I would say that the `place=farm` preset would first need a better name, perhaps something like "Farm Settlement" or "Individually Named Farm"?!

**Comment by AntMadeira** [comment]
That's a good point, but there are plenty of other examples like that. I think this is mainly a question for translators.
For example: for building=farm, the translation should be "farm building"; for landuse=farmyard, it should be "farm yard". 
The preset should appear, nevertheless.

Another example (not quite the same, but it fits to show there are alternatives): "bank" and" bench" have the same name in Portuguese (banco), and the translation has also the same name, but the different icons makes all the difference and allows mappers to distinguish presets easily.

**Comment by tyrasd** [comment]
Good point regarding the need for distinct icons in case where presets have similar or equal names. It could help here as well to distinguish the farm "place" from a farmyard or farm building. But in this instance it could also be tricky to find a suitable distinct icon for the place. Perhaps by combining the generic "map marker" icon which is used for a generic `place` with the farm icon?! Or alternatively base it of the icon for `place=hamlet`/`place=locality`, which use a triangle shape (but I have to admit that I'm not sure what exactly the triangle is supposed to represent :see_no_evil: ).

**Comment by matkoniecz** [comment]
"Named Farm"?

"Individually Named Farm"?

@1ec5 - ping?

**Comment by matkoniecz** [comment]
> I have to admit that I'm not sure what exactly the triangle is supposed to represent

generic unspecific feature? Match place symbology on paper maps?

**Comment by 1ec5** [comment]
Has it ever been settled that `place=farm` is really the right tag for a farm (in the sense of a complex on an individual private property) versus a `landuse=farmland` that may or may not be named? It seems problematic to require a name for `place=farm`. We don’t semantically require a name for other values such as `place=square` or `place=island`. If we accept that `place=farm` is the correct tagging, then “Farm” is the only possible English name for the preset, even if non-native speakers might confuse it with similarly named raw tags.

**Comment by matkoniecz** [comment]
I understood/misunderstood it to be specifically about named farms, isolated farm without name (or farm without name in a village) would not get it

similarly how large city may be for historic reasons considered a single named city or multiple named cities, see say https://www.openstreetmap.org/#map=12/50.2845/19.0070

<img width="1441" height="637" alt="Image" src="https://github.com/user-attachments/assets/ce199fa7-a720-495a-8f56-1bcaa1f064e2" />

City border goes between those two buildings

https://www.google.com/maps/place/Piechy,+Katowice/@50.2739075,18.9587579,3a,90y,199.08h,104.05t/data=!3m7!1e1!3m5!1sLZ4dTqoMO7LTjl0nZOmcOg!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fcb_client%3Dmaps_sv.tactile%26w%3D900%26h%3D600%26pitch%3D-14.046316191268417%26panoid%3DLZ4dTqoMO7LTjl0nZOmcOg%26yaw%3D199.07865140581788!7i13312!8i6656!4m6!3m5!1s0x4716cde944541167:0xb406fd6d3b2da97f!8m2!3d50.2733913!4d18.9598535!16s%2Fg%2F1vgqbt_b?hl=en&entry=ttu&g_ep=EgoyMDI2MDYyMy4wIKXMDSoASAFQAw%3D%3D

<img width="979" height="904" alt="Image" src="https://github.com/user-attachments/assets/0ddd089b-bdc4-4968-900c-44b262c64af5" />

https://www.openstreetmap.org/relation/2425990#map=19/50.273783/18.959489

<img width="1210" height="852" alt="Image" src="https://github.com/user-attachments/assets/9ccc1336-02ea-40a3-aece-be0fce227ba6" />

**Comment by matkoniecz** [comment]
and for what it is worth https://wiki.openstreetmap.org/wiki/Tag:place=farm has "An individually named farm." and from https://wiki.openstreetmap.org/w/index.php?title=Tag:place%3Dfarm&action=history it was not me who added it there

**Comment by AntMadeira** [comment]
Farms, especially as an isolated area with residential buildings and land around, are usually considered a place (as locality or isolated_dwelling are). In Portugal, for example, farms in the South, with vasts expansions of land, are considered places and have well known toponymic names. This should not be used to small farms which happen to have a name.

**Comment by matkoniecz** [comment]
> This should not be used to small farms which happen to have a name.

maybe https://wiki.openstreetmap.org/wiki/Tag:place=farm  should be changed then as it claims otherwise (or maybe you are wrong, no idea)

either way it should be probably discussed elsewhere whether OSM Wiki is wrong and how to change it if it is in fact wrong

**Comment by AntMadeira** [comment]
This paragraph from the wiki "A farm can be a part of a human settlement ([place](https://wiki.openstreetmap.org/wiki/Key:place)=* like hamlet, village, town) which has buildings and land for farming. Usually there are a few central buildings around a [landuse](https://wiki.openstreetmap.org/wiki/Key:landuse)=[farmyard](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dfarmyard) with [landuse](https://wiki.openstreetmap.org/wiki/Key:landuse)=[farmland](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dfarmland) or [landuse](https://wiki.openstreetmap.org/wiki/Key:landuse)=[meadow](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dmeadow) surrounding it." pretty much describes my idea of `place=farm`.

**Comment by matkoniecz** [comment]
oh sorry, then I misread your comment

**Comment by 1ec5** [comment]
> This should not be used to small farms which happen to have a name.

For what it’s worth, I think this might be consistent with the understanding in the U.S. as well, though we haven’t had a discussion about it in a very long time. `landuse=farmland` `name=*` is used about 70% more than `place=farm` (not to mention `landuse=orchard`, `landuse=vineyard`, etc.). Unlike in Europe, [only 36%](https://qlever.dev/osm-planet/ND16sn) of `place=farm` comes with `name=*`.

At a glance, some farms are dual-tagged with both. I suspect the ones tagged just `place=farm` are mostly industrial farms or ranches whose names are more useful for wayfinding (appearing on destination signs along the road), whereas most farm names are only posted on a small sign by the entrance, often as vanity.

**Comment by matkoniecz** [comment]
> [only 36%](https://qlever.dev/osm-planet/ND16sn) of place=farm comes with name=*

I would argue that it is tagging mistake and start from checking whether large part was imported / added by few people.

> landuse=farmland name=* is used about 70% more than place=farm

I expect this to be fairly rare in US, but how many of those are names of specific fields?

**Comment by 1ec5** [comment]
> I would argue that it is tagging mistake and start from checking whether large part was imported / added by few people.

Glancing at Ohio, I think this usage is pretty organic, but overall `place=farm` doesn’t seem popular here. By the way, many `landuse=farmland` started out as `landuse=farm` back when that was the correct tag for an individual farmstead but later got mass-migrated over.

The Farmland preset name suggests that it’s for broader agricultural landuse, i.e., the countryside. American English speakers gravitate toward it for both farmsteads and the farm fields within them because there aren’t any obvious alternatives. (Many farm fields are supposed to be Meadow, but this is very unintuitive.)

> I expect this to be fairly rare in US, but how many of those are names of specific fields?

Farmers hardly ever name individual fields. I’ve seen it at some demonstration farms at universities, or at farms that regularly host corn mazes. As a very rough estimate, [1,766](https://qlever.dev/osm-planet/x5LRJz) have something like “farm” in the name while only [427](https://qlever.dev/osm-planet/1FRiJW) have something like “field” in the name (many of them just “Field”).

**Comment by matkoniecz** [comment]
On taking look at data: I would say that 1200 nameless farms across USA do change so much much given overall tag use, and would not consider it as blocker given overall use of place=farm

if you or someone else plans to dispute wiki claims via some discussion or proposal - let me know and I can pause associated PR and wait for it to happen

but in this case it seems to me that wiki is right and can be followed

**Comment by 1ec5** [comment]
The [`place=farm`](https://wiki.openstreetmap.org/wiki/Tag:place%3Dfarm) documentation says to only use it within a larger settlement, or `place=isolated_dwelling` otherwise. This is completely nonsensical in a U.S. context, so I get a strong sense that there’s a cultural difference in what is meant by “farm” (or “ranch”). In the U.S., it’s always a single property, always originally outside of any settlement, though sometimes a city has built up around the one last remaining farm. A larger farm typically has a name for marketing purposes: you see the name at the grocery store. A smaller farm usually doesn’t have a name, but sometimes farmers name their farm for vanity, just like some of my neighbors have named their houses.

Some large farms are useful for wayfinding, namely, some of the [ranches out in West Texas that show up on destination signs](https://community.openstreetmap.org/t/should-emojis-be-used-in-tags/101823/35). Those are so rare that we don’t have a word for them. “Individually Named Farm” doesn’t seem to capture the rural/urban distinction documented on the wiki. “Notable Urban Farm” maybe? If we can’t resolve this difference, then I suggest hiding the `place=farm` preset in the U.S. or splitting it to give it a different name somehow.

This may also be related to [the recent forum discussion](https://community.openstreetmap.org/t/appetite-for-a-place-apartments-tag/145290) about `landuse=residential` `residential=apartments` being reserved for named apartment complexes.

**Comment by matkoniecz** [comment]
> This is completely nonsensical in a U.S. context, so I get a strong sense that there’s a cultural difference in what is meant by “farm” (or “ranch”).

I would interpret it differently - as "farm" is partially duplicating more established `place=isolated_dwelling` it may be preferable to use `place=isolated_dwelling` where appropriate and use `place=farm` where no other more established `place=` tag is validly applying

Even Poland has some standalone farms (in fact, one still active is relevant to my family history).

**Comment by matkoniecz** [comment]
I guess `place=farm` may be also appropriate for standalone named farm where noone lives? If there are such places. (I bet they exist)

**Comment by AntMadeira** [comment]
> I guess `place=farm` may be also appropriate for standalone named farm where noone lives? If there are such places. (I bet they exist)

Yes, they exist. The wiki doesn't state that someone needs to live there. Some of them may have already been inhabited, and others may be still have residential buildings. As long as the place itself has a farm with some central buildings and a land around, plus a well established name, I would consider it to be a `place=farm`.
Just a side note: in Portugal, even the military maps have most of these places mapped.

**Comment by 1ec5** [comment]
Here’s what the wiki has to say about it:

> an **individually named** farm, often including an area where a family of farmers resides … A farm can be a **part of a human settlement** (`place=*` like hamlet, village, town) which has buildings and land for farming. … If the farm is not part of a bigger settlement (regarding structure not administration) and the farm and comprising settlement itself is not **bigger than defined for isolated dwellings** in your country, use `place=isolated_dwelling`.

Maybe it’s poorly worded? It gives me the impression of a large agricultural development that is part of a larger populated place. I can think of an occasional [nursery](https://www.openstreetmap.org/way/34228115) or [outdoor museum](https://www.openstreetmap.org/way/191704264) that fits this definition, ignoring the problematic implications for place hierarchy, but the vast majority of individually named farms in the U.S. wouldn’t fit any part of the definition. Something like [Wittmeyer White Oak Valley Farm](https://www.openstreetmap.org/way/573587563) is too remote, and about average size for a farm or other property in the area.

A quick discussion [in OSMUS Slack](https://osmus.slack.com/archives/C2VJAJCS0/p1785188555737539) shows most mappers use named `landuse=farmland` and either didn’t know about `place=farm` or are confused about what it would be used for. I’m afraid that just wordsmithing a preset name won’t resolve the issue without a broader discussion on the forum. My best guess at the wiki’s position would be something like “Farm Settlement”. The broader use that @AntMadeira describes would be “farm”, but it seems very arbitrary to restrict it to named farms.

@matkoniecz and @AntMadeira: in your cultures, what would you call a single agricultural property, consisting of one or more fields/pastures and optionally a farmyard, ranging in size from 1 to say 600&nbsp;hectares, that does _not_ have a name? How would you tag it, if at all? I don’t dispute that the farms you’re describing are important enough to map, but it seems like we have a gap in the other direction, so adding this preset with an ambiguous name is going to dilute whatever definition you care about.

**Comment by matkoniecz** [comment]
"single agricultural property" - in Polish? gospodarstwo / gospodarstwo rolne?

"consisting of one or more fields/pastures and optionally a farmyard, ranging in size from 1 to say 600 hectares, that does not have a name" - still applicable, no special name for such case according to my nonexpert knowledge. Though I think it would definitely have farmyard.

note that in Poland individually named farms are rare, unless referred to by family name (and even that is disappearing, together with gradual disappearance of absurdly tiny farms)

> How would you tag it, if at all?

https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dfarmyard - and would likely put name on it if named (I would not put it on `landuse=farmland` as name in Poland, if present, would likely refer to habitable part, `landuse=farmland` would be likely micromapped and in chunks)

if I would remember about `place=farm` I would likely use it then if it is named place

> just wordsmithing a preset name won’t resolve the issue without a broader discussion on the forum

do you mean global one? To check is wiki definition matching actual use?

or US-specific forum?

are you interested in starting such discussion? From what I checked OSM Wiki definition matches actual use so I was not planning on starting one (I have queue of ones where OSM Wiki claims seem to be wrong or I think that other action needs to be taken, so I am not planning to start ones where someone else see a problem and I do not)

**Comment by 1ec5** [comment]
We have two problems that would benefit from a global discussion (doesn’t matter who starts it):

* Some of you seem to disagree with the wiki about the scope of `place=farm` – its relationship to other places. This affects whether “Individually Named Farm” would be an appropriate name for the preset, before even getting into the issue of how Nominatim handles the tag.
* If `landuse=farmland` is not for a “farm” (as it’s known in English) and `place=farm` must be named, then we lack a tag for the usual case of an identifiable farm that goes unnamed. How is that case being mapped, if not as `landuse=farmland`?

If possible, we should resolve these issues before introducing new presets that potentially introduce more confusion among laypeople. I think we want to avoid a repeat of the mess that forced us to deprecate `landuse=farm` in favor of `landuse=farmland`.

English-speakers doesn’t call `landuse=farmyard` a “farm”. We’d call that a “farmyard” or “farmstead”. (The latter can also refer to the whole farm.) In the anglophone world, farmyards are virtually never named specifically, only the farm as a whole. The [1.8%](https://taginfo.geofabrik.de/north-america:us/tags/landuse=farmyard#combinations) of named U.S. farmyards probably have a lot of tagging errors among them, including descriptive names, duplicates of surrounding farms, and misclassified farms or community gardens.

**Comment by matkoniecz** [comment]
> > [only 36%](https://qlever.dev/osm-planet/ND16sn) of place=farm comes with name=*

> I would argue that it is tagging mistake and start from checking whether large part was imported / added by few people.

Right now it is 36.68% vs 87.86% globally. And I started looking into data and contacting local mappers - and in cases I got reaction nameless `place=farm` was considered as a mistake or unfinished mapping.

**Comment by 1ec5** [comment]
The stat I linked that you quoted was just for the U.S., not globally. I followed up with [a question in Slack](https://osmus.slack.com/archives/C2VJAJCS0/p1785188555737539) and got consistent responses that `place=farm` is unknown or undesirable according to what “farm” means in the U.S. So that presents us with a terminology problem for the default locale, regardless of how well the tag fits globally and regardless of whether it should be named. At least the preset needs to exclude the U.S. to avoid confusion with the thing we call a farm here.

**Comment by matkoniecz** [comment]
> The stat I linked that you quoted was just for the U.S., not globally

yes, that is why I also added global one :)

> At least the preset needs to exclude the U.S. to avoid confusion with the thing we call a farm here.

that may work
